src/components/navigation/www/navigation.jsx

```diff
@@ -241,7 +241,8 @@ var Navigation = React.createClass({
                                 </a>
                             </li>,
                             <li className="link right account-nav" key="account-nav">
-                                <a className="user-info" href="#" onClick={this.handleAccountNavClick}>
+                                <a className={this.state.accountNavOpen ? 'user-info open' : 'user-info'}
```

Comment:
I think it might be more straightforward to do this before the return and use the `classes` library. That way, it doesn't need to be an either/or, but dependent on the presence of an extra class.